ui/src/app/auth/auth.guard.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable }           from '@angular/core';
+import { Router, CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot }  from '@angular/router';
+import { AuthService }          from './auth.service';
```

Comment:
Most examples show a blank line between angular/rxjs imports and app specific imports. That may help with some of the alignment here too. I'm no angular wiz and this is just personal preference, so take this comment with a grain of salt.